src/cr/cube/collator.py

```diff
@@ -10,15 +10,16 @@
 base vectors and inserted vectors, respectively.
 """
 
-import collections
 import sys
 
 import numpy as np
 
+from collections import OrderedDict
```

Comment:
I'm sure this is purely subjective, but I prefer to always import Python library modules whole and then refer to members with qualification, like `collections.OrderedDict`, so this one was made this way on purpose.